tests/test_sshfsmount.cpp

```diff
@@ -59,10 +67,52 @@ struct SshfsMount : public mp::test::SftpServerTest
         return request_exec;
     }
 
+    auto make_exec_to_check_commands(const command_vector& commands, std::string::size_type& remaining, command_vector::const_iterator& next_expected_cmd, std::string& output, bool& invoked)
+    {
+        auto request_exec = [&commands, &remaining, &next_expected_cmd, &output, &invoked](ssh_channel, const char* raw_cmd) {
+            std::string cmd{raw_cmd};
+            mp::utils::trim_end(cmd);
```

Comment:
Do we need to trim this? I feel like this may hide some extra whitespace that we shouldn't be executing anyway.

Reply:
You're right, it's unnecesary. I tested it.